engine/debug/debug_party.asm

```diff
@@ -20,7 +12,9 @@ SetIshiharaTeam:
 	call AddPartyMon
 	jr .loop
 
-IshiharaTeam:
+DebugNewGameParty: ; unreferenced (non-debug)
+	; Then producer Tsunekazu Ishihara used an Exeggutor for play testing.
```

Comment:
Why would you remove the direct quote and citation comment?

Reply:
Wouldn't this be better documented [here](https://github.com/pret/pokered/wiki/Trivia) after the change? I would go as far as removing the comment altogether.

Reply:
Eh, I don't mind having background info cited directly in the asm, when it applies to specific lines of the code/data. See also the comment in mew.asm. And the wiki isn't quite as authoritative as the asm itself, anyone can add trivia even if it's niche or vague or pointless or outright wrong.

Reply:
In that case, I'll review and reintroduce the information.